src/lambda_codebase/initial_commit/bootstrap_repository/adf-build/shared/helpers/terraform/get_accounts.py

```diff
@@ -38,6 +39,12 @@ def main():
         with open('accounts_from_ous.json', 'w', encoding='utf-8') as outfile:
             json.dump(accounts_from_ous, outfile)
 
+    if TARGET_TAGS:
+        print("filtering by tags")
```

Comment:
Could you remove this or change it to a `LOGGER` statement?

Reply:
Resolved with commit 646a3b485afe3fb56af23d20a8ccecca259b846b